datasets/il/mod_crypto/crawler.py

```diff
@@ -0,0 +1,89 @@
+from zavod import Context
+from zavod import helpers as h
+import csv
+
+
+def crawl(context: Context):
+    src = context.fetch_resource("source.csv", context.data_url)
+    with open(src, "r") as f:
+        reader = csv.DictReader(f)
+        for row in reader:
+            # Create a Person entity if present in the data
+            names = [
+                name
+                for name in [
+                    row.get("English Name"),
+                    row.get("Name Hebrew"),
+                    row.get("Name Arabic"),
+                ]
+                if name != ""
+            ]
+            if len(names) > 0:
+                person = context.make("Person")
```

Comment:
should we initialise person to None or set it to None in an else clause or is the persona associated with all the accounts below their name? If it's the latter case, perhaps a comment to that effect is a good idea to show it's intentional.

Reply:
Oooh, good point. I'll init to None, I think the person is only tied to the accounts in their row.